src/components/WorkPoolQueueStatusArray.vue

```diff
@@ -28,9 +28,10 @@
   const maxWorkQueues = 50
   const api = useWorkspaceApi()
 
+  const options = useInterval({ interval: 30000 })
   const workPoolQueuesSubscription = useSubscription(
     api.workPoolQueues.getWorkPoolQueues,
-    [props.workPool.name, { limit: maxWorkQueues + 1 }],
+    [props.workPool.name, { ...options, limit: maxWorkQueues + 1 }],
```

Comment:
I don't think this is the correct placement for the options. In fact, I don't think the limit piece is correct either? 
<img width="593" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/6200442/a5a89c84-be5b-4d62-acb8-fcbcfe148176">